microsoftgraph/msgraph-cli-archived
# Feat/rename mg to msgraph

## Overview

Renames mg to msgraph

### Demo

![image](https://user-images.githubusercontent.com/12011447/126042273-872a9d16-cad8-4b62-bb7b-99ed3acd6969.png)

### Notes

The `mg` command is used by used by "MicroGnu Emacs" on macos so we have to change the name of our command to avoid collision.

## Testing Instructions

* Install these changes
* Run `msgraph -h`
* Observe that you get a  list of command groups


Fixes #164 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/msgraph-cli/pull/165)
